app/routes.js

```diff
@@ -21,6 +21,7 @@ const restrictToStripeAccountContext = require('./middleware/stripe-setup/restri
 const restrictToSwitchingAccount = require('./middleware/restrict-to-switching-account')
 const uploadGovernmentEntityDocument = require('./middleware/multer-middleware')
 const inviteCookieIsPresent = require('./middleware/invite-cookie-is-present')
+const loginAfterRegister = require('./middleware/login-after-register.middleware')
```

Comment:
Agree with the middleware naming -- as it's a one line middelware I wonder if it's a little clearer to just inline here in routes 
```javascript
app.get(
  registerUser.logUserIn,
  passport.authenticate('localDirect', { failureRedirect: user.logIn }),
  userIsAuthorised,
  rootController.get
)
```

I'm not entirely convinced by this and could see pros and cons either way.

Reply:
The main advantage I can see of it living in its own file is it has a bit more of a descriptive name of what it does. Although maybe the strategy could be something named better than "localDirect" and that would fix it?

The disadvantage of having its own middleware file is you get to that file and it's just calling another middleware - which is a bit confusing.

Do you think inlining and renaming the strategy to "loginAfterRegistration" would be a good way to do it?

Reply:
oh yeah sounds like a clean middle ground 👍 I think the 'local' part has some semantic meaning in the context of strategies so maybe we could keep it, something like `localStrategyLoginDirectAfterRegistration`? Bit of a mouthful but I think it keeps the context and hopefully is much clearer as to what the strategy is for.